### DISCUSSION TITLE
Tuya Smart Life EU WiFi Double Curtain Blind Switch Integration

### DISCUSSION THREAD

**Discussion by Levanterman** [make-all/tuya-local#865] [General]
Hi 
I recently bought the [Tuya Smart Life EU WiFi Double Curtain Blind Switch](https://www.aliexpress.com/item/1005003080348426.html?spm=a2g0o.order_detail.order_detail_item.4.47d0f19cIPFxOJ)

I am not wiring any blinds into it, i just wanted to use the entities in HomeAssistant to trigger some existing RF blinds that i have . but when i connected this device to SmartLife and then Tuya HomeAssistant, it says its 'Unsupported'. I then added it to Tuya Local and it detects it as a Curtain Switch, which is perfect ! But it just detects the left 3 buttons and not the Double set. 

Is there any information/Logs/API values I can look up to assist with the double integration please?

Thanks
  
Nige.

**Comment by make-all** [comment]
Yes, when you added it, there should have been a log message showing all the dps that the device is reporting.  Or now that you have it connected, you can download the device diagnostics from Settings/Devices which should have the same info.
And if you have a developer account on iot.tuya.com, you may be able to get some detailed information about the dps labels and ranges from the API Explorer "Query Things Data Model" api function.

**Reply by Levanterman** [reply]
Thank you, I will delete it and re-add it, and gather all that information and submit a device request. I have just been reading about the local tuya device request process, thanks for replying.
